docs/releases/3.1.txt

```diff
@@ -344,6 +344,8 @@ Models
 * The new :class:`~django.db.models.F` expression ``bitxor()`` method allows
   :ref:`bitwise XOR operation <using-f-expressions-in-filters>`.
 
+* The ``sql_flush`` method in MySQL DatabaseOperations is faster now.
+
```

Comment:
What about

```suggestion
* The command:`sql_flush` command now generates more efficient SQL for table containing
  a few rows on MySQL by using ``DELETE`` statements instead of ``TRUNCATE``.
```

I think another mention should be added to the `Tests` section

```
* :class:`django.test.TransactionTestCase` teardown is now faster on MySQL due to
  command:`sql_flush` improvements on this backend. As a side effect the latter doesn't
  automatically reset sequences on teardown anymore. Enable
  :attr:`django.test.TransactionTestCase.reset_sequences` if your tests require such feature.
```

Reply:
Thank you for your suggestion. I added your suggestion in `Tests` section at a52ab51638.